improver/psychrometric_calculations/hail_size.py

```diff
@@ -235,32 +429,59 @@ def temperature_after_saturated_ascent_from_CCL(
         )
         return t_2
 
-    def dry_adiabatic_drop_to_ccl(
-        self, ccl_pressure: Cube, temperature_at_268: Cube, pressure_at_268: Cube
+    @staticmethod
+    def dry_adiabatic_descent_to_ccl(
+        ccl_pressure: Cube, temperature_at_268: Cube, pressure_at_268: Cube
     ) -> np.ndarray:
-        """Calculates a dry adiabatic drop from the pressure of the environment at
-        268.15K to the cloud condensation level."""
+        """Calculates the temperature due to a dry adiabatic descent from the pressure of the environment at
+        268.15K to the cloud condensation level pressure.
+
+        Args:
+            ccl_pressure
+                Cube of cloud condensation level pressure
+            temperature_at_268
+                Cube of the temperature of the environment at 268.15K
+            pressure_at_268
+                Cube of the pressure of the environment at 268.15K
+        Returns:
+            Cube of temperature after the dry adiabatic descent
+        """
 
         t_dry = dry_adiabatic_temperature(
             temperature_at_268.data, pressure_at_268.data, ccl_pressure.data
         )
         return t_dry
 
-    def get_hail_size(self, vertical: np.ndarray, horizontal: np.ndarray):
+    def get_hail_size(self, vertical: np.ndarray, horizontal: np.ndarray) -> np.ndarray:
         """Uses the lookup_table and the vertical and horizontal indexes calculated
         to extract and store values from the lookup nomogram. Masked data points or
-        if vertical or horizontal values are negative lead to a hail_size of 0."""
+        if vertical or horizontal values are negative lead to a hail_size of 0.
+
+        Args:
+            vertical
+                An n dimensional array containing the values used to calculate the vertical indexes
+            horizontal
+                An n dimensional array containign the values used to calculate the horizontal
```

Comment:
```suggestion
                An n dimensional array containing the values used to calculate the horizontal
```

Reply:
Done. One day I'll work out how to spell.